test/e2e/limit_range.go

```diff
@@ -61,7 +61,11 @@ var _ = Describe("LimitRange", func() {
 		Expect(err).NotTo(HaveOccurred())
 		for i := range pod.Spec.Containers {
 			err = equalResourceRequirement(expected, pod.Spec.Containers[i].Resources)
-			Expect(err).NotTo(HaveOccurred())
+			if err != nil {
+				// Print the pod to help in debugging.
+				Logf("Pod %+v does not have the expected requirements", pod)
```

Comment:
Would it make more sense for the error returned by `equalResourceRequirement` to have the necessary information?